src/lib/swapper/api.ts

```diff
@@ -274,3 +280,33 @@ export interface Swapper<T extends ChainId, MaybeUnknownNetworkFee extends boole
    */
   getTradeTxs(tradeResult: TradeResult): Promise<Result<TradeTxs, SwapErrorRight>>
 }
+
+export type UnsignedTx = SignTx<keyof ChainSignTx>
```

Comment:
Not required yet but we might want to revisit the vernacular here, this has ties to trade, sure it's a Tx but it's mostly a trade and that's the way we should probably envision it?